rdr_service/dao/bq_questionnaire_dao.py

```diff
@@ -47,45 +104,91 @@ def make_bqrecord(self, p_id, module_id, latest=False, convert_to_enum=False):
             logging.info('Generator: ignoring questionnaire module id [{0}].'.format(module_id))
             return None, list()
 
-        qnans = self.ro_dao.call_proc('sp_get_questionnaire_answers', args=[module_id, p_id])
-        if not qnans or len(qnans) == 0:
-            return None, list()
-
-        bqrs = list()
```

Comment:
Note: this block has moved to line 157 in the new version, but logically remains the same